UNRELEASED_TOKENS.md

```diff
@@ -0,0 +1,52 @@
+# Guide
+
+Read [#2093](https://github.com/hoprnet/hoprnet/issues/2093) for context.
+
+## Words
+
+| word            | description                                                                |
+| --------------- | -------------------------------------------------------------------------- |
+| HoprDistributor | a live smart contract on **mainnet** that users can claim HOPR tokens      |
+| CLAIMANT_ADDR   | an **ethereum address** that can claim HOPR tokens through HoprDistributor |
+| HOPR_ID         | a **HOPR id** of a HOPR node                                               |
+
+## Process
+
+1. User sends an email to tech@hoprnet.org with subject `Link my HOPR node`, and a message containing their `HOPR_ID` and the signature of `Link my Ethereum address to HOPR node address <HOPR_ID>` that was signed using `CLAIMANT_ADDR`. Example:
+
+```
+16Uiu2HAmQBZA4TzjKjU5fpCSprGuM2y8mpepNwMS6ZKFATiKg68h
+<signature of "Link my Ethereum address to HOPR node address 16Uiu2HAmQBZA4TzjKjU5fpCSprGuM2y8mpepNwMS6ZKFATiKg68h>
+```
```

Comment:
Can you provide an example of the resulting signature & data & metadata from MEW signer? IMO that would make everyone's life pretty easy.

Reply:
done